geany.nsi.in

```diff
@@ -124,13 +125,17 @@ Section "!Program Files" SEC01
 
 	SetOutPath "$INSTDIR\bin"
 	File "${RESOURCEDIR}\bin\Geany.exe"
-	File "${RESOURCEDIR}\bin\Geany*.dll"
+	File "${RESOURCEDIR}\bin\*Geany*.dll"
```

Comment:
This is odd too, but I guess one build system generates `libgeany-0.dll` and the other `geany-0.dll`?